contracts/pm/ERC20TicketBroker.sol

```diff
@@ -2,9 +2,9 @@ pragma solidity ^0.4.25;
 // solium-disable-next-line
 pragma experimental ABIEncoderV2;
 
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 import "./TicketBroker.sol";
-
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 // JUST AN EXAMPLE
 contract ERC20TicketBroker is TicketBroker {
```

Comment:
Missing a `using SafeMath for uint256`?

Reply:
Your EDIT is spot on. I removed the imports and using statements in subtypes.